### PR TITLE
Reverse Words

### PR DESCRIPTION
**Pull request by SKAUL05**

### CODE REVIEW

File: strings/reverse_words.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created by sarathkaul on 18/11/19
+
+
+def reverse_words(input_str: str) -> str:
+    """
+    Reverses words in a given string
+    >>> sentence = "I love Python"
+    >>> reverse_words(sentence) == " ".join(sentence.split()[::-1])
```
Comment (cclauss) [inline strings/reverse_words.py]: Let's also keep the other test to help the reader understand...
```python
>>> reverse_words(sentence)
'Python love I'
```

Comment (SKAUL05) [reply strings/reverse_words.py]: Okay Cool

File: strings/reverse_words.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    True
+    >>> sentence = "I love Python"
```
Comment (cclauss) [inline strings/reverse_words.py]: __sentence__ is already defined on line 7 so I think we can remove line 10.

File: strings/reverse_words.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    >>> reverse_words(sentence)
+    'Python love I'
+    """
+    input_str = input_str.split(" ")
+    new_str = list()
+
+    for a_word in input_str:
+        new_str.insert(0, a_word)
+
+    return " ".join(new_str)
```
Comment (chttrjeankr) [inline strings/reverse_words.py]: I just got the idea that `return " ".join(input_str[::-1])` would have probably worked the same way.

Comment (cclauss) [reply strings/reverse_words.py]: Line 8

Comment (chttrjeankr) [reply strings/reverse_words.py]: I noticed it being used as a doctest, and NOT the actual function. :sweat_smile: Hence, the comment.

**Pull request merged**